bundles/org.openhab.binding.atlona/src/main/java/org/openhab/binding/atlona/internal/pro3/AtlonaPro3PortocolHandler.java

```diff
@@ -1088,7 +1088,7 @@ private class NormalResponseCallback implements SocketSessionListener {
 
         @Override
         public void responseReceived(String response) {
-            if (response == null || response.isEmpty()) {
+            if (response.isEmpty()) {
```

Comment:
This class is not `@NonNullByDefault`, so I'm wondering if this change introduces risk of NPE? I have not checked usages etc., so just question.

Reply:
It is only used once in SocketChannelSession and is wrapped with a null check there. I removed the null check here to eliminate the redundant null check warning.
![socketchanelsession](https://user-images.githubusercontent.com/26532542/182863147-8b720a64-d702-4c2c-8717-3a02e9d09bb8.jpg)

Reply:
Much better not to call with null. Thanks for clarifying.